buildSrc/src/main/groovy/io.micronaut.build.internal.pulsar-module.gradle

```diff
@@ -7,6 +7,19 @@ repositories {
 }
 
 dependencies {
+    api mn.micronaut.messaging
+    api libs.micronaut.serde.support
+    api(libs.managed.pulsar) {
+        exclude group: 'io.netty'
+        exclude group: 'com.fasterxml.jackson.module'
+        exclude group: 'com.google.code.gson'
+    }
+
+    implementation(libs.managed.pulsar) {

```

Comment:
:x: it doesn't make sense to have the same dependency in both `api` and `implementation`: it should only be in `api`.